elastic/kibana
# [Security Solution] Remove graphQL host

## Summary

This PR is to remove graphQL endpoint for host entirely. Before this PR we've replace every endpoint of host with search strategy except from host details.

Steps to verify:
1. Go to host page.
2. Enable the dev tool of Chrome and switch to network tab, filter `graphql`, make sure nothing appears.
3. Click any host name on the page and land on host details page. Wait for the page to load. Observe the network panel in dev tool, make sure no graphql request appears.
